src/common/KeyboardAvoider.js

```diff
@@ -51,7 +75,9 @@ export default function KeyboardAvoider(props: Props): Node {
       behavior={behavior}
       contentContainerStyle={contentContainerStyle}
       // See comment on this prop in the jsdoc.
-      keyboardVerticalOffset={keyboardVerticalOffset}
+      keyboardVerticalOffset={
+        keyboardVerticalOffset + (compensateOverpadding ? -bottomInsetHeight : 0)
```

Comment:
At this spot we're abusing the `keyboardVerticalOffset` prop of `KeyboardAvoider` -- what we're doing doesn't match its intended interface. So a comment to acknowledge that would be helpful, letting the reader know the mismatch is intentional and they're not just missing something. Like this, say:
```suggestion
      keyboardVerticalOffset={
        keyboardVerticalOffset
        // A negative term here reduces the bottom inset we use for the child,
        // when the keyboard is open, so that the keyboard covers its bottom padding.
        + (compensateOverpadding ? -bottomInsetHeight : 0)
```